### ISSUE TITLE
not converting numbers will leading zeros

### ISSUE THREAD

**Issue by avilella** [harelba/q#171]
Hi, I have a case where a .csv has a table with numbers like: 032907365 that are getting converted to 32907365 when doing:
```
avilella@avilellaM710t:~$ echo -e "col\n032907365\n"
col
032907365

avilella@avilellaM710t:~$ echo -e "col\n032907365\n" | q -O -H -d ',' "SELECT col  FROM - " 2>/dev/null
col
32907365

avilella@avilellaM710t:~$ echo -e "col\n032907365\n" | q -O -H -d ',' "SELECT CAST(col as TEXT)  FROM - " 2>/dev/null
CAST(col as TEXT)
32907365

```
How can I switch off that conversion process?

**Comment by avilella** [comment]
Resolved with q version 1.7.x and:

```
echo -e "col\n032907365\n" | q --as-text -O -H -d ',' "SELECT col  FROM - " 2>/dev/null
col
032907365
```

**Comment by harelba** [comment]
thanks @bitti and @avilella for handling :)
